index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined">
+    <link rel="stylesheet" href="style.css">
+    <title>Alpha portifolio</title>
+</head>
+<body>
+    <header class="header">
+        <p class="logo">Alpha Portifolio</p>
+        <i class="fa fa-bars"></i>
+    </header>
+    <main>
+        <div class="name-intro">
+            I’m Alpha <br>
+            Glad to see <br> you!
+        </div>
+        <div class="personal-info">
+            I’m a software developer! I can help you build a product , feature or 
+            website Look through some of my work and experience! If you like 
+            what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
+        <div class="connect"><a href="#">LET'S CONNECT</a> </div>
+        <ul class="social">
+            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-medium" aria-hidden="true"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="#">✌🏿</a></li>
+        </ul>
```

Comment:
- [ ] It would be better to wrap the headline section in a section tag, this will make your website more semantic 🤗